site-service-client/src/main/java/uk/ac/ox/ndph/mts/siteserviceclient/SiteServiceClient.java

```diff
@@ -79,11 +79,11 @@ public SiteResponseDTO createEntity(final SiteDTO site,
         return Arrays.asList(requestExecutor.sendBlockingGetRequest(webClient, uri, SiteDTO[].class, authHeaders));
     }
 
-    public List<SiteDTO> getAssignedSites(final Consumer<HttpHeaders> authHeaders) {
+    public List<String> getParentSiteIds(final String siteId, final Consumer<HttpHeaders> authHeaders) {
```

Comment:
Is getAssignedSites() implemented somewhere? Again, this is a method we do need.